t/extract-version.t

```diff
@@ -648,20 +648,20 @@ foreach my $test_case (@modules) {
     # We want to ensure we preserve the original, as long as it's legal, so we
     # explicitly check the stringified form.
     {
-      local $TODO = !defined($got) && ($test_case->{TODO_code_sub} || $test_case->{TODO_scalar});
+      local $TODO = !defined($got) && ($test_case->{TODO_code_sub} || $test_case->{TODO_scalar}) || undef;
       isa_ok($got, 'version') or $errs++ if defined $expected_version;
     }
 
     if (ref($expected_version) eq 'CODE') {
-      local $TODO = $test_case->{TODO_code_sub};
+      local $TODO = $test_case->{TODO_code_sub} || undef;
```

Comment:
This line shouldn't need to be changed - `$test_case->{TODO_code_sub}` is never defined-but-false.